src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -653,6 +653,66 @@ public static IntPtr GetHINSTANCE(Module m)
             return GetExceptionForHRInternal(errorCode, errorInfo);
         }
 
+#if TARGET_WINDOWS
+        private static readonly Guid IID_ISupportErrorInfo = new Guid([
+                    0x60, 0x3D, 0x0B, 0xDF, // Data1: 0xDF0B3D60 (little-endian)
```

Comment:
This is atypical Guid formatting. Would it be better to use the typical Guid constructor that takes int32, int16, int16, .... ?